src/briefcase/commands/convert.py

```diff
@@ -808,3 +796,20 @@ def __call__(
                 project_overrides=parse_project_overrides(project_overrides),
                 **options,
             )
+
+
+def find_changelog_filename(base_path):
+    """Search for a valid changelog file in the specified directory
+    and return its format name if it exists, otherwise return None"""
```

Comment:
The formatting of docstrings is significant:
- the first line should be a brief description; ideally a single line.
- Parameters are annotated with `:param:`
- Return values are annotated with `:returns:`
```suggestion
    """Find a valid changelog file in a given directory.
    
    :param base_path: The directory to search
    :returns: The filename of the changelog that was found; None if a changelog
        with a known name could not be found.
    """
```